control-plane-operator/controllers/hostedcontrolplane/kas/aws_kms.go

```diff
@@ -216,6 +217,10 @@ func buildKASContainerAWSKMSTokenMinter(image string) func(*corev1.Container) {
 			fmt.Sprintf("--token-file=%s", path.Join(awsKMSVolumeMounts.Path(c.Name, kasVolumeAWSKMSCloudProviderToken().Name), "token")),
 			fmt.Sprintf("--kubeconfig=%s", path.Join(awsKMSVolumeMounts.Path(c.Name, kasVolumeLocalhostKubeconfig().Name), KubeconfigKey)),
 		}
+		c.Resources.Requests = corev1.ResourceList{
+			corev1.ResourceCPU:    resource.MustParse("10m"),
```

Comment:
where are these values coming from?

Reply:
These are the same values used for the `token-minter`  container here https://github.com/openshift/hypershift/blob/main/hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go#L2201-L2205